staff/robert/task-app/task-app-client/src/index.js

```diff
@@ -0,0 +1,48 @@
+require('dotenv').config()
+
+const { taskApi } = require('./task-api-client')
+const express = require('express')
+const bodyParser = require('body-parser')
+const axios = require('axios')
```

Comment:
the constant axios is necessary?. I think you don't use it in this file

Reply:
done